pkg/network/opencontrail/cni/contrail-server.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2010 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+type vmResp struct {
+	Vm   string `json:"vm"`
+	Ip   string `json:"ip"`
+	Plen int    `json:"plen"`
+	Gw   string `json:"gw"`
+	Dns  string `json:"dns"`
+}
+
+type statusResp struct {
+	Status string `json:"status"`
+}
+
+var addr int = 3
```

Comment:
Want it to be global variable so that we keep giving incremental addresses everytime. This is only a test-code. Moved contrail-server.go to test/ directory in subsequent commit